GSA/digitalgov.gov
# Table: Fixed border issue

## Summary

Fixed table styles and improved code quality and readability.

### Preview

[Link to Preview]()



### Solution

Removed and updated custom table styles that were conflicting base USWDS designs.

This was found 11 times through out the site but have been checked and not major visual regressions were found but all tables are now looking like the USWDS table design. 

Below is a table to compare the live site to what the tables will look like with this update. 

| Live     | Preview |
| ----------- | ----------- |
|    [about page - live](https://digital.gov/about/)   |   [about page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug/about/)     |
| [hcd feedback page - live](https://digital.gov/guides/hcd/design-concepts/feedback/#content-start)   | [hcd feedback page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//guides/hcd/design-concepts/feedback/#content-start)       |
| [hcd methods page - live](https://digital.gov/guides/hcd/discovery-concepts/methods/#content-start)   | [hcd methods page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//guides/hcd/discovery-concepts/methods/#content-start)       |
| [understanding data guide - live](https://digital.gov/guides/site-scanning/understand-the-data/)   | [understanding data guide - live](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//guides/site-scanning/understand-the-data/)         |
| [only table not broken - live](https://digital.gov/2023/03/07/a-happy-compromise-between-people-first-and-plain-language/)   | [only table not broken - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//2023/03/07/a-happy-compromise-between-people-first-and-plain-language/)        |
| [content page - live](https://digital.gov/2023/06/13/how-to-choose-the-right-metrics-for-your-intranet/)   | [content page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//2023/06/13/how-to-choose-the-right-metrics-for-your-intranet/)       |
| [content page - live](https://digital.gov/2023/11/15/defining-benchmarks-and-targets-for-intranet-kpis/)   | [content page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//2023/11/15/defining-benchmarks-and-targets-for-intranet-kpis/)       |
| [resource page - live](https://digital.gov/resources/an-introduction-to-analytics/)  | [resource page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//resources/an-introduction-to-analytics/)        |
| [resource page - live](https://digital.gov/resources/an-introduction-to-security-and-privacy-controls/)   | [resource page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//resources/an-introduction-to-security-and-privacy-controls)       |
| [resource page - live](https://digital.gov/resources/introduction-to-translation-technology/)   | [resource page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//resources/introduction-to-translation-technology/)        |
| [resource page - live](https://digital.gov/resources/service-delivery-transition-plan/)   | [resource page - preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-table-bug//resources/service-delivery-transition-plan/)       |



### How To Test

1. Visit the the links in the table above and verify that tables are now designed as USWDS tables. 
---

### Dev Checklist
